corehq/apps/userreports/rebuild.py

```diff
@@ -126,7 +126,7 @@ def add_columns(engine, raw_diffs, table_names):
             col.table = None
             changes[table_name].append({
                 'type': DiffTypes.ADD_COLUMN,
-                'value': col.name
+                'item_name': col.name
```

Comment:
where is this change used?

Reply:
This get's stored in the `diff` column of the `DataSourceAction` table so I wanted it to be consistent with `SimpleDiff` which is now also being stored there.